.bcr/presubmit.yml

```diff
@@ -8,15 +8,21 @@ tasks:
     bazel: ${{ bazel }}
     build_targets:
     - '@apple_support//lib/...'
-  run_tests:
-    name: "Run test targets"
-    platform: "macos_arm64"
-    bazel: ${{ bazel }}
-    build_targets:
-    - '@apple_support//lib/...'
-    test_targets:
-    - '@apple_support//test/...'
-    - '--'
-    # Needs visionOS SDK
-    - '-@apple_support//test:binary_visionos_arm64_simulator_test'
-    - '-@apple_support//test:binary_visionos_device_test'
+
+bcr_test_module:
+  module_path: "."
+
+  matrix:
+    bazel: ["7.x", "rolling"]
```

Comment:
This isn't already inherited from the top-level declaration, right?

Reply:
I don't believe so based on the docs.

Reply:
Confirmed it's needed.